receiver/prometheusreceiver/factory_test.go

```diff
@@ -49,6 +50,28 @@ func TestFactoryCanParseServiceDiscoveryConfigs(t *testing.T) {
 }
 
 func TestMultipleCreate(t *testing.T) {
+	factory := NewFactory()
+	cfg := factory.CreateDefaultConfig().(*Config)
+	cfg.APIServer = &APIServer{
+		Enabled: true,
+		ServerConfig: confighttp.ServerConfig{
+			Endpoint: "localhost:9090",
+		},
+	}
+	set := receivertest.NewNopSettings()
+	firstRcvr, err := factory.CreateMetrics(context.Background(), set, cfg, consumertest.NewNop())
+	require.NoError(t, err)
+	host := componenttest.NewNopHost()
+	require.NoError(t, err)
+	require.NoError(t, firstRcvr.Start(context.Background(), host))
+	require.NoError(t, firstRcvr.Shutdown(context.Background()))
+	secondRcvr, err := factory.CreateMetrics(context.Background(), set, cfg, consumertest.NewNop())
+	require.NoError(t, err)
+	require.NoError(t, secondRcvr.Start(context.Background(), host))
+	require.NoError(t, secondRcvr.Shutdown(context.Background()))
+}
+
+func TestMultipleCreateWithAPIServer(t *testing.T) {
```

Comment:
I think this test should be named `TestMultipleCreate`, and the one above should be `TestMultipleCreateWithAPIServer` (they are switched)